dxr/plugins/rust/tests/test_smoke/test_smoke.py

```diff
@@ -0,0 +1,23 @@
+# Index a huge file containing as much of Rust as I could think of. Check that
+# index finishes without crashing. This is also a good test for manual
+# experimentation.
+
+from nose import SkipTest
+from nose.tools import eq_,ok_
+from dxr.testing import DxrInstanceTestCase
+import os
+
+# We'll also need rust libs to be in LD_LIBRARY_PATH, but there's not an easy
+# way to test for that.
+if 'RUSTC' not in os.environ:
+    raise SkipTest
+
+class RustTests(DxrInstanceTestCase):
+    """Test indexing of Rust projects"""
+
+    def test_smoke(self):
+        """Test the index exists and didn't crash."""
+
+        response = self.client().get('/')
```

Comment:
You don't even need anything here. If it crashes, there'll be an exception, and the explosion will register as a test failure (test error, technically). For comparison, see my test_build_failure.py:

```python
    def test_nothing(self):
        """A null test just to make the setup method run"""
```